run

```diff
@@ -43,7 +43,7 @@ Commands
 ##
 
 # Define docker images versions
-dev_image="canonicalwebteam/dev:v1.6.4"
+dev_image="canonicalwebteam/dev:v1.6.5"
```

Comment:
The code uses `Object.fromEntries` which requires Node 12. This wasn't an issue before because the test wasn't exercising that code path.